### PR TITLE
allows authentication via database password #285

### PR DESCRIPTION
**Pull request by rvalyi**
implements my part of https://github.com/OCA/server-tools/issues/285

### CODE REVIEW

**Comment by gurneyalex** [comment]
@nhomar do you have a waffle.io configuration automatically adding labels on your behalf? If so, could you remove this repository from your config?

**Comment by nhomar** [comment]
all, sorry for the invoncenience, it is solved now. (I didn't realize of that) thanks).

File: auth_admin_passkey/model/res_users.py [hunk 1]
```diff
@@ -132,6 +134,13 @@ def check_credentials(self, cr, uid, password):
                     cr, uid, password)
                 return True
             except exceptions.AccessDenied:
-                return self.check_credentials(cr, SUPERUSER_ID, password)
+                icp_obj = self.pool['ir.config_parameter']
+                allow_db_pwd = safe_eval(icp_obj.get_param(
+                    cr, SUPERUSER_ID, 'auth_admin_passkey.db_pwd', 'False'))
+                if allow_db_pwd and password and\
+                        password == config['db_password']:
+                    return True
```
Comment (gurneyalex) [inline auth_admin_passkey/model/res_users.py:142]: could you extract these 6 lines to a separate method so that the behavior can be customized?

**Comment by gurneyalex** [comment]
small refactoring suggested. 

overall :+1:

**Pull request closed without merging**